r/R/embedding.R

```diff
@@ -1,79 +1,85 @@
-#runEmbedding
-#Returns a list of x,y coordinates for each cell.
-#req is the request.
+# runEmbedding
+# Returns a list of x,y coordinates for each cell.
+# req is the request.
 #
-#Req$body has:
-#type = type of embedding, supported umap, pca and tsne.
-#config = config list.
+# Req$body has:
+# type = type of embedding, supported umap, pca and tsne.
+# config = config list.
```

Comment:
Why wasn't this formatted well in the first place? Is everyone using the same linter for R?

Reply:
We now all use the same linter, which is the Rstudio linter so this shouldn't happen. We have a ticket to add automatic linting to PRs